orchestrator/node-red/src/index.js

```diff
@@ -51,7 +51,7 @@ module.exports = class NodeAPI {
     });
 
     app.get('/nodes', (req, res) => {
-      const expectedResponseType = req.accepts(['application/json', 'text/html']);
+      // const expectedResponseType = req.accepts(['application/json', 'text/html']);
```

Comment:
It might be: https://expressjs.com/pt-br/4x/api.html#req.accepts

Once this "accepts" method does nothing on the request itself (it only returns the best match for a particular format), and as the returned variable was not being used, it should have no side-effects.